ledger/accountdb.go

```diff
@@ -1970,7 +1976,7 @@ type persistedValue struct {
 func (qs *accountsDbQueries) lookupKeyValue(key string) (pv persistedValue, err error) {
 	err = db.Retry(func() error {
 		var v sql.NullString
-		err := qs.lookupKvPairStmt.QueryRow(key).Scan(&pv.round, &v)
+		err := qs.lookupKvPairStmt.QueryRow([]byte(key)).Scan(&pv.round, &v)
```

Comment:
Was the []byte cast done to prevent unicode interpretation or something like that?